### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by schoeberl** [chisel-uvm/chisel-verify#4]
When running sbt test we get following error:
```
[error] /Users/martin/source/chisel-uvm/chisel-verify/src/test/scala/coverage/CoverageDB.scala:60:48: value strip is not a member of String
[error]             coverIdPortMap update (dutPortName.strip(), dutPort)
[error]                                                ^
[error] one error found
[error] (Test / compileIncremental) Compilation failed
```

**Comment by Dobios** [comment]
Works fine on my machine, what version of scala are you using?

**Comment by Dobios** [comment]
Fixed in commit [a490d58](https://github.com/chisel-uvm/chisel-verify/commit/a490d588df6b64a1f6b0299b4da4da4d431dff16)
